core/src/main/java/feign/Contract.java

```diff
@@ -31,15 +31,35 @@
   /**
    * Called to parse the methods in the class that are linked to HTTP requests.
    *
+   * @deprecated Use {@link #parseAndValidateMetadata(Class)} instead
    * @param targetType {@link feign.Target#type() type} of the Feign interface.
    */
   // TODO: break this and correct spelling at some point
-  List<MethodMetadata> parseAndValidatateMetadata(Class<?> targetType);
+  @Deprecated
+  default List<MethodMetadata> parseAndValidatateMetadata(Class<?> targetType){
```

Comment:
I would say to take out the typo method

Reply:
I have ammended the commit to not pollute the master branch